packages/plugin-ext/src/main/browser/plugin-ext-deploy-command.ts

```diff
@@ -28,7 +28,7 @@ export class PluginExtDeployCommandService implements QuickOpenModel {
 
     public static COMMAND: Command = {
         id: 'plugin-ext:deploy-plugin-id',
-        label: 'Plugin: Deploy a plugin\'s id'
+        label: 'Plugin: Deploy Plugin by Id'
```

Comment:
why this change is not being asked to be reviewed by members of plug-ins work ?

Reply:
For now I'm -1 due to all screenshots/tests including the previous name of the command

Reply:
Hi @benoitf, yes sorry about that I did mean to have someone from plugin review but I also felt the change of a label was minor to be able to be consistent with the rest of theia. Also while working on a different PR I cam across this command and it didn't make immediate sense to me at what its functionality was as we do not deploy a plugin's id but rather a plugin by id. 

I'm not quite sure as to what screenshots/tests you are referring to but if you'd like me to discard this particular change then I can do so.

Reply:
@benoitf 
> For now I'm -1 due to all screenshots/tests including the previous name of the command

I guess by that you mean that the Che documentation that includes screenshots of the current label and it would be confusing to change it without updating your screenshots?

Reply:
yes but let's be consistent then